### PR TITLE
Home work l5

### PR DESCRIPTION
**Pull request by AxLShd0w**
Please check my homework

### CODE REVIEW

File: src/com/ua/HomeWork/l5/GreaterNumber.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.ua.HomeWork.l5;
+
+import java.util.Scanner;
+
+public class GreaterNumber {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("A = ");
+        int A = scanner.nextInt();
```
Comment (oleksii-kapustnyk) [inline src/com/ua/HomeWork/l5/GreaterNumber.java]: no capital letters/ should be 'int a'

File: src/com/ua/HomeWork/l5/GreaterNumber.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        System.out.print("B = ");
+        int B = scanner.nextInt();
+
+        if (A > B){
+            System.out.println("Number A is bigger than Number B");
+        } else {
+            System.out.println("Number B is bigger than Number A");
+        }
```
Comment (oleksii-kapustnyk) [inline src/com/ua/HomeWork/l5/GreaterNumber.java:20]: what if a == b?

File: src/com/ua/HomeWork/l5/GreaterNumber.java (hunk 1 above)
Comment (AxLShd0w) [reply src/com/ua/HomeWork/l5/GreaterNumber.java]: I have corrected the errors. Please check

File: src/com/ua/HomeWork/l5/GreaterNumber.java (hunk 2 above)
Comment (AxLShd0w) [reply src/com/ua/HomeWork/l5/GreaterNumber.java:20]: Add one more if in solution

**Review by oleksii-kapustnyk — APPROVED** [review]

**Pull request merged**